pkg/networkmanager/interfaces.js

```diff
@@ -1445,6 +1445,12 @@ PageNetworking.prototype = {
         $("#networking-add-bridge").click($.proxy(this, "add_bridge"));
         $("#networking-add-vlan").click($.proxy(this, "add_vlan"));
 
+        $("#networking-add-team").hide();
+        cockpit.script("which rpmquery && rpmquery NetworkManager-team", { err: "ignore" }).
```

Comment:
> This also doesn't work on Debian or Ubuntu.

Well, it works in the sense that team support is not enabled, cleanly.  But yes, I'll look at what the code would be for Debian.